src/components/Panels/Shared/HelpButton.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+	<p v-if="helplink" class="btn-parent">
+		<button 
+			id="HelpButton" 
+			class="btn btn-primary btn-xs" type="button" 
+			@click="helpButtonClick"
+		>
+			&#9432 Help
+		</button>
+	</p>
+</template>
+
+<script lang="ts" setup>
+const props = defineProps({
+    data: { type: Object, default: undefined },
+})
+const obj = props.data
+var helplink = obj && obj.helplink
```

Comment:
just some tips: you can checkout JS Optional Chaining feature.

Reply:
Thanks, I implemented it for help link